scripts/ci/upload-build

```diff
@@ -1,22 +1,29 @@
 #!/bin/bash
 set -ex
 
-DIST_PATH=${DIST_PATH:-dist}
-BUILD_PREFIX=${BUILD_PREFIX:-oss}
+# configuration
+BUILD_VARIANT=${BUILD_VARIANT:-''}
+BRANCH_NAME=${BRANCH_NAME:-'master'}
+DIST_PATH=${DIST_PATH:-'dist'}
+PACKAGE_PATH=${PACKAGE_PATH:-'.'}
+GIT_SHA=${GIT_SHA:-$(git rev-parse --short HEAD)}
```

Comment:
We could try finding another name because of https://github.com/mesosphere/dcos-ui-plugins-private/pull/737/files#diff-ca24b89c55f8d23ebebff270bf89f180R89 it's not a git sha anymore but rather `SHA_SLUG`?